app/src/main/java/com/duckduckgo/app/privatesearch/PrivateSearchActivity.kt

```diff
@@ -73,6 +79,20 @@ class PrivateSearchActivity : DuckDuckGoActivity() {
                         newCheckedState = it.autoCompleteSuggestionsEnabled,
                         changeListener = autocompleteToggleListener,
                     )
+                    if (it.storeHistoryEnabled) {
```

Comment:
Maybe we can simplify all visiblity logic by having all of them as part of a viewgroup in the xml and just showing/hiding the viewgroup?

Reply:
I was assuming it wasn't possible to use a `ViewGroup` in a `CoordinatorLayout` but I'll have a look just in case